examples/vault/demo.load-sample-policies.sh

```diff
@@ -24,9 +24,9 @@ loadVault()
 
   # Use policy templates to create policy files.
   # The example below uses 3 different policies with the following constraints:
-  #  - all - uses cluster-region, cluster-name, namespace and images
-  #  - n - uses cluster-region, cluster-name, namespace
-  #  - r - uses cluster-region
+  #  - all - uses region, cluster-name, namespace and images
+  #  - n - uses region, cluster-name, namespace
+  #  - r - uses region
```

Comment:
Hmm it looks like a hierarchy here, 

like

region -> cluster-name -> namespace -> images

So maybe we can say that if you match "namespace" you match anything preceding? Just wondering if it would be easier to think about it that way?